dist/scripts/api-console.js

```diff
@@ -2882,13 +2882,25 @@
           $scope.responseDetails = false;
           $scope.response        = {};
 
+          var mockingServiceDetector = /(?:(?:mocksvc\.[a-z\.]*)|(?:[a-z]+\.anypoint\.))mulesoft\.com\/.*mocks\/([0-9a-zA-Z-]+)(\/(.+))?/;
+          var mockingService2Detector = /(?:(?:[a-z.]*anypoint\.mulesoft\.com)|(?:localhost:3000))\/mocking\/api\/v1\/links\/([0-9a-zA-Z-]+)(\/(.*))?$/;
+
```

Comment:
The case where mocking service 2 is called without using links is not being taken into account. I know that that case should never happen but, why not just checking the beginning of the URL? It would match for both cases